src/components/notebookRenderStrategy.tsx

```diff
@@ -42,6 +46,23 @@ export class NotebookRenderStrategy implements ExpandableNodeRenderStrategy {
 	}
 
 	getChildren(childrenLevel: number): JSX.Element[] {
+		if (typeof (this.notebook.apiHttpErrorCode) === 'number') {
+			const errorString = Strings.getError(this.notebook.apiHttpErrorCode);
+			return [
+				<li role='status' aria-live='polite' aria-label={errorString} className='progress-row'>
+					<div>{errorString}</div>
+				</li>
+			];
+		}
+
+		if (this.notebook.needsToFetchChildren) {
```

Comment:
Better name: IsFetchingChildren?

Reply:
I think in the context of the notebook, needs to fetch makes more sense. As it is not fetching until the notebook is expanded.